lib/Slackbot_worker.js

```diff
@@ -338,6 +337,10 @@ module.exports = function(botkit, config) {
     };
 
     bot.send = function(message, cb) {
+		if (message.ephemeral) {
```

Comment:
I chose to retain a single send function, and just check if the message should be ephemeral or not

Using send like this also preserves access to these messages from send middlewares!